juriscraper/lib/importer.py

```diff
@@ -34,10 +34,10 @@ def find_all_attr_or_punt(court_id):
             # juriscraper.opinions.united_states.federal_appellate.ca1,
             # therefore, we add it to our list!
             module_strings.append(court_id)
-        except ImportError as e:
-            # Something has gone wrong with the import
-            print("Import error: %s" % e)
-            return []
+        #except ImportError as e:
```

Comment:
Ah yes, this was so it would hard fail if a scraper failed. What was happening was the failures with scrapers weren't surfacing and the test was "passing".

Are you saying just remove it?